<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>340</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Thread Start</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txbLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgdTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
